front/src/containers/post/dateRoute.js

```diff
@@ -0,0 +1,27 @@
+import { useParams } from 'react-router-dom'
+import { RouteMatch } from '../../components/pages/routeMatch'
+import { NotFound } from '../../components/pages/notFound'
+
+export function DateRouteContainer() {
+
+  const PAGE_TITLES = {
+    DATE_IN_PAST: 'Date in the past'
+  }
+
+  const { date } = useParams()
+
+  const matchDateFrom = (input) => {
+    const match = input.match(/^\d{4}[\-](0[1-9]|1[0-2])[\-](0[1-9]|[1-2][0-9]|3[0-1])$/)
```

Comment:
Leading zero in a month and day are not supported by this RegExp, this can be changed upon request.